README.md

```diff
@@ -130,6 +130,27 @@ Tables to export. inspections, inspection_items, templates, template_permissions
 env: `IAUD_EXPORT_TABLES`\
 default: All tables
 
+#### `export.incremental`
+Update actions, action_assignees, inspections, inspection_items and templates tables incrementally
+> flag: `--incremental-update`\
+env: `IAUD_EXPORT_INCREMENTAL`\
```

Comment:
Its not related to this PR, But are we using these env's anywhere ?

Reply:
There is potentially that end users are. But I will include the breaking change in the release notes.